src/schemas/ErrorSchema.json

```diff
@@ -0,0 +1,15 @@
+{
+    "type": "object",
+    "properties": {
+      "statusCode": {
+        "type": "integer"
```

Comment:
non-blocking: why`integer` and not `number`?

Reply:
The http responses are 400, 404 and not 400.5 I considered defining it as an integer, but if you think number is better I'll change it